features/command_line_interface/exit_status.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Return the correct exit status codes
+  In order to use Rubycritic in Continuous Integration environments
+  Rubycritic returns the exit status code properly
+
+  Scenario: Status indicates a success when analyse a clean file
+    Given the clean file 'clean.rb'
+    When I run rubycritic clean.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates a success when analyse a smelly file
+    Given the smelly file 'smelly.rb'
+    When I run rubycritic smelly.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates a success when analyse a clean file with the minimum score option
+    Given the clean file 'clean.rb'
+    When I run rubycritic --minimum-score 100 clean.rb
+    Then the exit status indicates a success
+  Scenario: Status indicates an error when analyse a smelly file with the minimum score option
+    Given the smelly file 'smelly.rb'
```

Comment:
Can we make this step more expressive by actually telling the caller what score this file has explicitly?
Thinking of something like

>> Given the smelly file 'smelly.rb' with a score of 115